<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerPage</class>
 <widget class="QWidget" name="PlayerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1538</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="palyerHead" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 0, 0);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>13</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="vedioTitle">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 170, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="sysBtnBox" native="true">
        <property name="minimumSize">
         <size>
          <width>112</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>112</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 170, 0);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>36</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>31</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="minBtn">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(170, 255, 0);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="quitBtn">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(0, 0, 127);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="playerMid" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0, 85, 0);</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="playerBottom" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>186</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>186</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 85, 0);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="videoSlider" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 0, 127);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="playControl" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(170, 85, 127);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>30</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="playTimeBox" native="true">
           <property name="minimumSize">
            <size>
             <width>126</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>126</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 0, 0);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="playBtn">
              <property name="minimumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="videoDuration">
              <property name="text">
               <string>28:22/32:33</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="bulletScreenBox" native="true">
           <property name="minimumSize">
            <size>
             <width>1116</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>1116</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(85, 0, 0);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>30</number>
            </property>
            <property name="leftMargin">
             <number>30</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="bulletScreenBtn">
              <property name="minimumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="bulletScreenText"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="speedBox" native="true">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(85, 85, 0);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="speedBtn">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>21</height>
               </size>
              </property>
              <property name="text">
               <string>倍速</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="volumeBox" native="true">
           <property name="minimumSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(85, 170, 0);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="volumeBtn">
              <property name="minimumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>26</width>
                <height>26</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="videoInfoBox" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 255, 127);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="videoInfo" native="true">
           <property name="minimumSize">
            <size>
             <width>310</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>310</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(85, 0, 127);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="playNumBox" native="true">
           <property name="minimumSize">
            <size>
             <width>114</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>114</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(255, 0, 127);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="likeNumBox" native="true">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(85, 170, 127);</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="videoIntroduceBox" native="true">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 170, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
